pom.xml

```diff
@@ -15,7 +15,7 @@
 
   <properties>
     <nuxeo.target.version>11.*</nuxeo.target.version>
-    <nuxeo.image.version>11.x</nuxeo.image.version>
+    <nuxeo.image.version>${nuxeo.platform.version}</nuxeo.image.version>
```

Comment:
Yes it is used for the maven docker build, and retrieved from maven when building the image from the pipeline: i just simplified the usage so that i don't have to duplicate replacement of "11.x" when releasing.
Said differently: i'm not relying on 11.x anymore for this version (even if we could assume the latest docker image pushed matches the latest nuxeo-parent pushed to nexus, i'd rather not assume it, especially when releasing).